tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -383,21 +382,71 @@ describe(__filename, () => {
     dispatchSpy.resetHistory();
 
     root.setProps({
-      addonType: ADDON_TYPE_OPENSEARCH,
+      addonType: ADDON_TYPE_STATIC_THEME,
       authorIds: [randomAuthorId2],
     });
 
     sinon.assert.calledWith(
       dispatchSpy,
       fetchAddonsByAuthors({
-        addonType: ADDON_TYPE_OPENSEARCH,
+        addonType: ADDON_TYPE_STATIC_THEME,
         authorIds: [randomAuthorId2],
         errorHandlerId: errorHandler.id,
         pageSize: String(numberOfAddons),
       }),
     );
   });
 
+  it('shows add-ons in header if no specific addonType translation found', () => {
+    const root = renderAddonsWithType({
+      addonType: 'unknown-type',
+      multipleAuthors: false,
+    });
+
+    expect(root.find(AddonsCard)).toHaveProp(
+      'header',
+      `More add-ons by ${fakeAuthor.name}`,
+    );
+  });
+
+  it('shows add-ons in header if no specific addonType translation found without More text', () => {
+    const root = renderAddonsWithType({
+      addonType: 'unknown-type',
+      showMore: false,
+      multipleAuthors: false,
+    });
+
+    expect(root.find(AddonsCard)).toHaveProp(
+      'header',
+      `Add-ons by ${fakeAuthor.name}`,
+    );
+  });
+
+  it('shows add-ons in header if no specific addonType found with multiple authors', () => {
+    const root = renderAddonsWithType({
+      addonType: 'unknown-type',
+      multipleAuthors: true,
+    });
+
+    expect(root.find(AddonsCard)).toHaveProp(
+      'header',
+      'More add-ons by these developers',
+    );
+  });
+
+  it('shows add-ons in header if no specific addonType found with multiple authors and without More text', () => {
+    const root = renderAddonsWithType({
+      addonType: 'unknown-type',
+      showMore: false,
+      multipleAuthors: true,
+    });
+
+    expect(root.find(AddonsCard)).toHaveProp(
+      'header',
+      'Add-ons by these developers',
+    );
+  });
```

Comment:
Same here, please move the test cases below "should dispatch a fetch action if forAddonSlug is updated"

Reply:
Yeah. Sure thing :)

Reply:
Looks like these test cases need to be moved below (the diff still shows additions/deletions)